Common/include/adt/CADTNodeClass.hpp

```diff
@@ -0,0 +1,48 @@
+/*!
+ * \file CADTNodeClass.hpp
+ * \brief Class for storing the information needed in a node of an ADT.
+ * \author E. van der Weide
+ * \version 7.0.5 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2020, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#include "../basic_types/datatype_structure.hpp"
+
+/*!
+ * \class CADTNodeClass
+ * \brief  Class for storing the information needed in a node of an ADT.
+ * \author E. van der Weide
+ */
+struct CADTNodeClass {
```

Comment:
This file, and the CBBoxTargetClass only contain structs but I still split them into their own header files. These don't have any cpp files associated with them